conans/test/utils/tools.py

```diff
@@ -610,6 +610,16 @@ def export(self, ref, conanfile=GenConanfile(), args=None):
         tmp.revision = rrev
         return tmp
 
+    def alias(self, source_target):
```

Comment:
I think it would be easier to define two params:

```suggestion
    def alias(self, source, target):
    """A beautiful docstring here"""
```

Given the fact that it's a helper and it's doing more than a simple `self.run(f"new alias ...")`, I guess two params and a docstring would be fine.

Reply:
Done